src/test/cypress/fixtures/exercise/file-upload/template.json

```diff
@@ -0,0 +1,30 @@
+{
+    "mode": "INDIVIDUAL",
+    "includedInOverallScore": "INCLUDED_COMPLETELY",
+    "numberOfAssessmentsOfCorrectionRounds": [
+        {
+            "inTime": 0,
+            "late": 0
+        }
+    ],
```

Comment:
This should not be needed when creating a new exercise.
```suggestion
```

Reply:
Done :)